### PR TITLE
Add support for arm64.

### PR DESCRIPTION
**Pull request by dantwinkler**
* [x] I have read the [contribution documentation](https://github.com/electron-userland/electron-packager/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] The changes are appropriately documented (if applicable).
* [x] The changes have sufficient test coverage (if applicable).
* [x] The testsuite passes successfully on my local machine (if applicable).

Add support for arm64.  The support has been added to the libchromiumcontent project and to the electron project but an official release is not yet publicly available.  This change has been tested against a built arm64 version of electron (link below).

To test:

// Until official build is ready
export NPM_CONFIG_ELECTRON_MIRROR=https://github.com/6RiverSystems/electron/releases/download/ 

// Single test
node ./cli.js ./test/fixtures/basic --platform=linux --arch=arm64 --electron-version=1.7.0

Tested Release
https://github.com/6RiverSystems/electron/releases/tag/1.7.0

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ It generates executables/bundles for the following **target** platforms:
 
 * Windows (also known as `win32`, for both 32/64 bit)
 * OS X (also known as `darwin`) / [Mac App Store](http://electron.atom.io/docs/v0.36.0/tutorial/mac-app-store-submission-guide/) (also known as `mas`)<sup>*</sup>
-* Linux (for x86, x86_64, and armv7l architectures)
+* Linux (for x86, x86_64, armv7l, arm64 architectures)
```
Comment (malept) [inline readme.md]: Please put back the "and".

Comment (dantwinkler) [inline readme.md]: Fixed.

File: NEWS.md [hunk 2]
```diff
@@ -99,11 +99,11 @@
 ### Added
 
 * `win32metadata` option (#331, #463)
-* `linux` platform, `armv7l` arch support (#106, #474)
+* `linux` platform, `armv7l` or `arm64` arch support (#106, #474)
```
Comment (dantwinkler) [inline NEWS.md]: Reverted this until the PR is accepted.

File: readme.md (hunk 1 above)
Comment (dantwinkler) [inline readme.md]: Did the maintainers of the electron project discuss give any timeline for arm64 support?

File: .idea/workspace.xml [hunk 3]
```diff
@@ -0,0 +1,732 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (malept) [inline .idea/workspace.xml]: Please don't add this.

Add `.idea` to your global `.gitignore`.

**Comment by malept** [comment]
Since this PR has been out of date for a while and also needs a lot of work, I'm closing it in favor of another one, to be pushed shortly.

**Pull request closed without merging**